docker/helper.Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1
 ARG SOURCES_DIR=/usr/src/ipa
 FROM rust:latest as builder
-
+ARG SOURCES_DIR
```

Comment:
Not surprised, but Linux version has slightly different expectations (not sure why though)